build/cni/Makefile

```diff
@@ -29,7 +29,7 @@ all: build
 build:
 	mkdir -p $(OUTPUT_DIR)
 
-	docker run -it -v $(OUTPUT_DIR):/output golang:$(GOLANG_VERSION) /bin/bash -c "\
```

Comment:
can you explain how this fixes the issue? I don't understand what the `:Z` option does.

(I also imagine there are other instances where this might need to be fixed.)